### PR TITLE
Adding memory settings required to build akka #26265

### PR DESCRIPTION
**Pull request by nvollmar**

### CODE REVIEW

**Comment by nvollmar** [comment]
@raboof :)

**Comment by raboof** [comment]
OK TO TEST

**Review by raboof — APPROVED** [review]
I think this can improve the contribution experience :+1:

File: .sbtopts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-J-Xmx2048M
+-J-Xms512M
+-J-XX:MaxMetaspaceSize=512M
+-J-XX:+CMSClassUnloadingEnabled
```
Comment (johanandren) [inline .sbtopts]: Does this really do anything unless you also enable `UseConcMarkSweepGC`?

File: .sbtopts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+-J-Xmx2048M
+-J-Xms512M
+-J-XX:MaxMetaspaceSize=512M
```
Comment (johanandren) [inline .sbtopts]: Do we really need to limit metaspace?

**Review by johanandren — APPROVED** [review]
LGTM but isn't just the Xmx (and maybe Xms for faster startup) enough?

File: .sbtopts (hunk 2 above)
Comment (nvollmar) [reply .sbtopts]: sbt default for max metaspace is 256M, I had to increase that limit to get akka compiling

File: .sbtopts (hunk 1 above)
Comment (nvollmar) [reply .sbtopts]: Right, it does not do anything -> removed

**Comment by johanandren** [comment]
Fixes #26265

**Pull request merged**